tests/golden_files/block_left_new.txt

```diff
@@ -5,8 +5,9 @@ Traceback (most recent call last):
                                ^^^^^^^^^^
       73 |              for letter
                         ^^^^^^^^^^
-      74 |                in
+      74 | 
```

Comment:
This is actually a very difficult test to analyse visually. The content of this line ends with "74 | ". Notice the extra space at the end. When I was editing this "golden" text file with one editor that shall remain nameless, it would strip this ending space on that line, so that the test would fail.